src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -318,24 +322,24 @@ public void move(Vector spdVec, double rot) {
    * @param rateLimit whether or not to use slew rate limiting
    */
   private void move(Vector spdVec, double rot, boolean rateLimit) {
-    spdVec = spdVec.copy();
-
+    Vector spdCommanded = spdVec.copy();
     m_currentRotationRadians = rot;
 
+    // Adjust input based on max speed
+    spdCommanded.mult(DriveConfig.kMaxSpeedMetersPerSecond);
+
     if (rateLimit) {
-      spdVec = limitDirectionSlewRate(spdVec);
+      spdCommanded = limitDirectionSlewRate(spdVec);
       m_currentRotationRadians = m_rotLimiter.calculate(rot);
     }
 
-    // Adjust input based on max speed
-    Vector spdDelivered = spdVec.copy().mult(DriveConfig.kMaxSpeedMetersPerSecond);
```

Comment:
Why is this removed?

Reply:
I moved it to above the slew rate limiting to prevent a future bug where the slew rate limiting would apply to the proportional input (a range of -1 to 1) instead of the actual movement speed range (-max speed to max speed)